workitem_blackbox_test.go

```diff
@@ -84,6 +86,45 @@ func TestGetWorkItem(t *testing.T) {
 		t.Errorf("expected title %s, but got %s", "Updated Test WI", updated.Data.Attributes[workitem.SystemTitle])
 	}
 
+	// Reorder Tests
+
+	// when the item is moved between two items
+	_, result2 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload)
+	_, result3 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload)
+	payload3 := minimumRequiredReorderPayload()
+	payload3.Data.ID = updated.Data.ID
+	payload3.Data.Attributes = updated.Data.Attributes
+	_, reordered1 := test.ReorderWorkitemOK(t, nil, nil, controller, *result3.Data.ID, result2.Data.ID, &payload3)
+	assert.NotNil(t, reordered1.Data.Attributes[workitem.SystemCreatedAt])
+	assert.NotNil(t, reordered1.Data.Attributes[workitem.SystemOrder])
+
+	if reordered1.Data.Attributes["version"] != (updated.Data.Attributes["version"].(int) + 1) {
+		t.Errorf("expected version %d, but got %d", (updated.Data.Attributes["version"].(int) + 1), reordered1.Data.Attributes["version"])
```

Comment:
Please use this instead to avoid the boilerplate:

```go
assert.Equal(t, updated.Data.Attributes["version"].(int) + 1, reordered1.Data.Attributes["version"])
```